rest-api-spec/src/main/resources/rest-api-spec/test/search.inner_hits/10_basic.yml

```diff
@@ -45,8 +45,8 @@ setup:
 "Nested doc version and seqIDs":
 
     - skip:
-        version: " - 6.3.99"
-        reason:  "object notation for docvalue_fields was introduced in 6.4"
+        version: " - 6.99.99"
+        reason:  "Triggers warnings before 7.0"
```

Comment:
Not sure but should we have a test to check for warnings that runs on 6.4-7.0 versions for the mixed cluster bwc tests?

Reply:
A test that would have "6.3.99 - 6.99.99" as a version would never run since 7.0 is not included?

Reply:
hmmm I think I might have a slightly misunderstanding on how the mixed cluster tests work then as I thought we set up a mixed cluster (let say 5 modes with 3 as 6.x and 2 as 7.0) and then we ran the YAML tests against the 6.x nodes using the 6.x version to work out which tests to run. Though now that I'm writing this I realise that as long as the 6.x version tests that the warning is sent (which it does) it should be enough and we probably shouldn't have 7.0 effectively test somethign in 6.x.